### PR TITLE
[ISSUE:3088] format SQL 

### LINKED ISSUE #3088: [Volunteer] format  SQL to improve code readability in shenyu-admin.
### Description

Hi Community:
This PR is to format the current SQL, following the principle of right-aligning keywords (Uppercase) and left-aligning fields. I would like to make this task a good first issue so that new comers can get involved in handling the rest of the SQL file. Do you think it's a good idea?

for example:
before formatting:

```SQL
select id, name, age from user where id = 1
```

after formatting

```SQL
  SELECT id, 
         name, 
         age 
    FROM user
   WHERE id = 1
```

### Task List

- [x] 1. app-auth-sqlmap.xml @midnight2104 
- [x] 2. auth-param-sqlmap.xml @hutaishi
- [x] 3. auth-path-sqlmap.xml @[ShawnSiao](https://github.com/ShawnSiao)
- [x] 4. dashboard-user-sqlmap.xml @[Jingwugate](https://github.com/Jingwugate)
- [x] 5. data-permission-sqlmap.xml @[Mayank-KS](https://github.com/Mayank-KS)
- [x] 6. meta-data-sqlmap.xml @[selectbook](https://github.com/selectbook)
- [x] 7. permission-sqlmap.xml @[wklong](https://github.com/wklong)
- [x] 8. plugin-handle-sqlmap.xml [joker-zxs](https://github.com/joker-zxs)
- [x] 9. plugin-sqlmap.xml  @[zhangcly](https://github.com/zhangcly)
- [x] 10. resouce-sqlmap.xml @[nydia](https://github.com/nydia)
- [x] 11. role-sqlmap.xml @Albertsirius
- [x] 12. rule-condition-sqlmap.xml  @UniverseInHeart
- [x] 13. rule-sqlmap.xml @totalo
- [x] 14. selector-condition-sqlmap.xml @lanseBlue
- [x] 15. selector-sqlmap.xml @xiaheshun
- [x] 16. shenyu-dict-sqlmap.xml @shallwetalkge
- [x] 17. user-role-sqlmap.xml @erdengk 

Notice: input `about issue 3088` in your pr desc .

### PR DESCRIPTION
**Pull request by erdengk**
// Describe your PR here; eg. Fixes #issueNo

fix #3088

### CODE REVIEW

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 1]
```diff
@@ -27,46 +27,61 @@
...
 
     <select id="selectById" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from user_role
-        where id = #{id,jdbcType=VARCHAR}
+        SELECT
+               <include refid="Base_Column_List"/>
+          FROM user_role
+        WHERE id = #{id, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:41]: The keywords `WHERE` are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 2]
```diff
@@ -27,46 +27,61 @@
...
 
     <select id="findByUserId" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from user_role
-        where user_id = #{userId,jdbcType=VARCHAR}
+        SELECT
+               <include refid="Base_Column_List"/>
+          FROM user_role
+        WHERE user_id = #{userId, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:48]: The keywords `WHERE` are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 3]
```diff
@@ -27,46 +27,61 @@
...
+                     date_created,
+                     date_updated,
+                     user_id,
+                     role_id)
+               VALUES
+                    (#{id, jdbcType=VARCHAR},
+                     #{dateCreated, jdbcType=TIMESTAMP},
+                     #{dateUpdated, jdbcType=TIMESTAMP},
+                     #{userId, jdbcType=VARCHAR},
+                     #{roleId, jdbcType=VARCHAR})
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:63]: The keywords  are not right-aligned, fileds  are not left-aligned

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 4]
```diff
@@ -27,46 +27,61 @@
...
-        insert into user_role (id, user_id, role_id)
-        values
-        <foreach collection="userRoleList" index="index" item="userRole" separator=",">
-            (
-            #{userRole.id,jdbcType=VARCHAR},
-            #{userRole.userId,jdbcType=VARCHAR},
-            #{userRole.roleId,jdbcType=VARCHAR}
-            )
-        </foreach>
+        INSERT INTO user_role
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:67]: The keywords `INSERT INTO` are not right-aligned

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 5]
```diff
@@ -81,39 +96,37 @@
...
+                #{roleId, jdbcType=VARCHAR},
             </if>
         </trim>
     </insert>
 
     <delete id="delete" parameterType="java.lang.String">
-        delete from user_role
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM user_role
+        WHERE id = #{id, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:117]: The keywords  `WHERE` are not right-aligned

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 6]
```diff
@@ -81,39 +96,37 @@
...
     </delete>
 
     <delete id="deleteByUserId" parameterType="java.lang.String">
-        delete from user_role
-        where user_id = #{userId,jdbcType=VARCHAR}
+        DELETE FROM user_role
+        WHERE user_id = #{userId, jdbcType=VARCHAR}
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:122]: The keywords  `WHERE` are not right-aligned.

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml [hunk 7]
```diff
@@ -81,39 +96,37 @@
...
     </delete>
 
     <delete id="deleteByUserIdSet">
-        delete from user_role
-        where user_id in
-        <foreach collection="userIdSet" index="index" item="userId"
-                 open="(" separator="," close=")">
-            #{userId,jdbcType=VARCHAR}
+        DELETE FROM user_role
+        WHERE user_id in
```
Comment (midnight2104) [inline shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:127]: The keywords  `WHERE` are not right-aligned.

**Comment by erdengk** [comment]
@midnight2104 
I looked at the already merged  #3078  and  #3098 . And made changes to my pr.

File: shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml (hunk 3 above)
Comment (erdengk) [reply shenyu-admin/src/main/resources/mappers/user-role-sqlmap.xml:63]: image
I followed this merged pr to make the changes.

**Comment by yu199195** [comment]
@erdengk  used about to replace fix   issueNo

**Comment by erdengk** [comment]
> @erdengk used about to replace fix issueNo

ok

**Review by midnight2104 — APPROVED** [review]

**Pull request merged**